src/server/requestHandlers.js

```diff
@@ -338,6 +348,31 @@ sjrk.storyTelling.server.deleteSingleFileRecoverable = function (fileToDelete, d
     }
 };
 
+fluid.defaults("sjrk.storyTelling.server.clientConfigHandler", {
+    gradeNames: "kettle.request.http",
+    invokers: {
+        handleRequest: {
+            funcName: "sjrk.storyTelling.server.getClientConfig",
+            args: ["{arguments}.0", "{server}.options.globalConfig", "{server}.options.secureConfig"]
+        }
+    }
+});
+
+// Returns a collection of values which are "safe" to share
+// with the client side of the application
+sjrk.storyTelling.server.getClientConfig = function (request, globalConfig, secureConfig) {
+    var clientConfig = {
+        theme: globalConfig.theme || secureConfig.baseThemeName,
+        baseTheme: secureConfig.baseThemeName,
```

Comment:
As we discussed last Friday, as the client no longer needs to distinguish the base theme from other themes with the change of embedding "base" in component names, the server doesn't need to return `baseTheme: "base"` to the client.

Reply:
I kept it in there for the check in `sjrk.storyTelling.loadThemedPage`, since we don't want to load `base.js` or `base.css` (which don't exist) and we need something to compare against in order to make that evaluation.

Reply:
We should discuss if a header template should be added to handle the inclusion of css and js from theme directories which should also help to solve [SJRK-256 issue](https://issues.fluidproject.org/projects/SJRK/issues/SJRK-256).

Reply:
That sounds like a good idea, though I feel it might be a bit out of scope for this Jira. I agree that we should talk about it for sure :)

Reply:
I agree this can be addressed in a separate jira. Can you create one and link with SJRK-256? Thanks.

Reply:
[SJRK-264](https://issues.fluidproject.org/browse/SJRK-264) has been filed.